src/base/net/dnsupdater.cpp

```diff
@@ -37,8 +37,11 @@
 #include "base/net/downloadmanager.h"
 #include "base/version.h"
 
+using namespace std::chrono_literals;
 using namespace Net;
 
+const int IP_CHECK_INTERVAL = std::chrono::milliseconds(30min).count();
```

Comment:
Why it is `int` and not `std::chrono::milliseconds`?

Reply:
OK, changed.